vispy/color/colormap.py

```diff
@@ -508,7 +510,158 @@ def map(self, t):
                            np.sqrt(t))
 
 
+class _SingleHue(Colormap):
+    """A colormap which is solely defined by the given hue and value.
+
+    Given the color hue and value, this color map increases the saturation
+    of a color. The start color is almost white but still contains a hint of
+    the given color, and at the end the color is fully saturated.
+
+    Parameters
+    ----------
+    hue : scalar, optional
+        The number indicating the hue value, which refers to a "true" color,
+        without any shading or tinting. Must be in the range [0, 360]. Defaults
+        to 200 (blue).
+    saturation_range : array-like, optional
+        The saturation represents how "pure" a color is. Less saturation means
```

Comment:
It's inconsistent that this one is "The thing" while the other two are "The number indicating/representing the thing".